automated-comments: Added rust/reverse-string comments (#1151)
exercism/website-copy: automated-comments/rust/reverse-string/suggest_doing_bonus_test.md

Consider expanding the solution to pass the feature-gated `test_grapheme_clusters` test case.

> Understanding how non-ASCII symbols can be handled and how to use external crates, has great benefits down the line.

Reviewer: Reverse-string is the first exercise after hello-world, right? If so, it might be useful to add some links to pages detailing how to add external crates, and possibly even the crate itself or a hint on what the crate's name is.

Author: The hint on the crate name is already in the README: https://github.com/exercism/rust/blob/master/exercises/reverse-string/README.md#bonus

As for the learning link, how about this:
Understanding how non-ASCII symbols can be handled and how to [use external crates](https://doc.rust-lang.org/cargo/guide/dependencies.html), has great benefits down the line.

Reviewer: I like the link! And thanks for the crate name explanation.

> The potential benefits of the expansion is the increased learning in the non-ASCII symbols handling area and the additional training in using the external crates, if the need arises to use one.

Reviewer 2: This is _very_ wordy. Perhaps something simpler?

```suggestion
Understanding how non-ASCII symbols can be handled using external crates has great benefits down the line.
```

Author: Yes, you wording is much better.
Just a small moment - there are two items in the comment:

Understanding how non-ASCII symbols can be handled **and**  the usage of the external crates has great benefits down the line.

The student may try to reverse non-ASCII by hand - although daunting, but still a valid approach.

Reviewer 2: > The student may try to reverse non-ASCII by hand - although daunting, but still a valid approach.

 Hehe, but they probably _shouldn't_ right?

```suggestion
Understanding how non-ASCII symbols can be handled and how to use external crates has great benefits down the line.
```

Like this?

Author: Yes, lets leave it at that
